pyresample/geometry.py

```diff
@@ -1113,8 +1128,10 @@ def __init__(self, area_id, description, proj_id, projection, width, height,
         self.pixel_size_y = (area_extent[3] - area_extent[1]) / float(height)
         self.area_extent = tuple(area_extent)
         if CRS is not None:
-            self.crs = CRS(projection)
+            # self.crs = CRS(projection)
```

Comment:
Can we make self.crs a property that generates the crs on the fly from the internal wkt representation?

Reply:
That's what it is. This is commented out because this was the old way of doing it. See the below `_crs` property that is used and defined even further below. I can remove this comment which was left by accident.

Reply:
I'm removed the comment now.